### PR TITLE
chore: Add community info page

### PR DESCRIPTION
**Pull request by gaocegege**
https://deploy-preview-198--envd.netlify.app/community/community.html

### CODE REVIEW

**Comment by gaocegege** [comment]
PTAL

File: docs/.vitepress/config/sidebar/main.ts [hunk 1]
```diff
@@ -10,7 +10,6 @@ export const mainSidebar: DefaultTheme.Sidebar = {
         { text: 'Running Your First Environment', link:'/guide/build-envd'},
         { text: 'Using envd Environment', link:'/guide/ide'},
         { text: 'envd Concepts', link:'/guide/concepts'},
-        { text: 'Modularize your build', link:'/guide/modularize'},
```
Comment (kemingy) [inline docs/.vitepress/config/sidebar/main.ts:13]: Is this one hidden?

**Review by kemingy — COMMENTED** [review]
Why the preview is zh site instead of en?

**Comment by gaocegege** [comment]
> Why the preview is zh site instead of en?

Known issues for netlify.

File: docs/.vitepress/config/sidebar/main.ts (hunk 1 above)
Comment (gaocegege) [reply docs/.vitepress/config/sidebar/main.ts]: I added it back. It is by accident. Thanks for the review.

**Review by kemingy — APPROVED** [review]
/lgtm

**Pull request merged**